runtime/v2/app.go

```diff
@@ -111,3 +114,13 @@ func (a *App[T]) Close() error {
 func (a *App[T]) GetApp() *App[T] {
 	return a
 }
+
+// SetStoreKeys sets the store keys.
```

Comment:
can we remove this getter and replace by `appBuilder.app.storeKeys = storeBuilder.GetStoreKeys()` where it is used? not one should ever call this, so let's make it impossible.